presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -91,6 +101,10 @@ public OperatorStats(
             @JsonProperty("finishCpu") Duration finishCpu,
             @JsonProperty("finishUser") Duration finishUser,
 
+            @JsonProperty("weightedHashCollisions") double weightedHashCollisions,
+            @JsonProperty("weightedHashCollisionsSquared") double weightedHashCollisionsSquared,
```

Comment:
Actually, for all the hash collision properties, these are specific to only two operators so you should use the `setInfoSupplier` API on `OperatorContext` instead of adding these to `OperatorStats`